common4j/versioning/version.properties

```diff
@@ -1,4 +1,4 @@
 #Wed May 12 20:08:39 UTC 2021
-versionName=5.0.0
+versionName=5.0.1
```

Comment:
Just want to make sure this is right. Did we not end up needing a 5.0.2?

Reply:
This is correct.
For common8.0.1 there was no common4j release required, so it consumed 5.0.0.
Common 8.0.2 consumes common4j 5.0.1